### DISCUSSION TITLE
Dark mode code blocks on readthedocs

### DISCUSSION THREAD

**Discussion by ianhi** [pradyunsg/furo#380] [Potential Issue] [answered]
Hi,

I just switched to using furo (looks great!) and code blocks rendered fine locally, but when rendered on readthedocs.com (https://mpl-point-clicker.readthedocs.io/en/latest/) the code blocks don't seem to have gotten the approriate dark mode styling:

![image](https://user-images.githubusercontent.com/10111092/155026090-db8e1b28-8e76-4997-b17e-5094d96779a2.png)

I'm not doing much in my conf.py so it seems as though there is something I should be doing that I'm currently not. Any help much appreciated thanks!

**Comment by ianhi** [comment, marked as answer]
Ahh and eventually it seems to have resolved itself. Mysterious caching I suppose (I thought F5 refresh would be enough but I was wrong). Anyway thanks!

**Reply by pradyunsg** [reply]
So... Furo has fairly decent cache-busting for the files that it generates (various `furo*` files).

However, all the things that Sphinx generates the URLs for (like the pygments CSS) don't use any sort of cache-busting strategies; which means that they can get stale that way. You _might_ be able to do a `curl -X PURGE {url}` depending on your hosting provider, to make sure all your users have their caches refreshed.
